packages/gatsby-admin/src/pages/index.tsx

```diff
@@ -102,46 +148,28 @@ const Index: React.FC<{}> = () => {
   if (error) return <p>Oops something went wrong.</p>
 
   return (
-    <>
-      <h1>{data.npmPackageJson.value.replace(/^"|"$/g, ``)}</h1>
-      <h2>Plugins</h2>
-      <ul>
+    <Flex gap={7} flexDirection="column" sx={{ paddingY: 7, paddingX: 6 }}>
```

Comment:
Curious why you decided to use Theme UI in *addition* to Gatsby Interface – seems like this might be confusing for other people coming in to work on this...

Reply:
I'm not sure I follow, those two packages solve completely different problems? One is our design system component library, one is a CSS-in-JS library? 🤔

Reply:
I don't think they solve *completely* different problems, ideally a design system should provide virtually all of the styles we'd need to build out UI IMO. Curious to hear what @Elanhant's thoughts might be though (this is definitely not-blocking BTW)

Reply:
To be honest, I am not completely sure what is the question here 🙂

Reply:
My understanding, which could very well be wrong, was that Gatsby Interface was a design system/component library that could build out most of the UI without needing other libraries. If that's not the case, and it's intended to work alongside something like Theme UI, then that's all I'm asking about here

Reply:
I think your understanding would be right for Gatsby Interface v1 (correct me if I'm wrong, @fk). For now, however, it is just a library of the components used the most in Cloud, so it is inevitable to have cases where you'll have to resort to a 3rd party or custom solution (in which case, let me and/or Flo know, perhaps it warrants an addition to Gatsby Interface!).